### PR TITLE
Update icons to version 2.57

### PR DESCRIPTION
**Pull request by Jahnp**

### CODE REVIEW

**Comment by Jahnp** [comment]
Looks like there's some cleanup to do, both to fix the build and remove another logo I missed the first time around. Will update as soon as I can--disembarking for a long flight shortly.

File: src/data/icons.json [hunk 1]
```diff
@@ -54,6 +55,7 @@
   { "name": "AnalyticsReport", "unicode": "F1E1" },
   { "name": "AnchorLock", "unicode": "F511" },
   { "name": "Annotation", "unicode": "E924" },
+  { "name": "ApacheIvyLogo32", "unicode": "F5D1" },
```
Comment (mikewheaton) [inline src/data/icons.json]: Think you've called this icon out in your PR comment, but just want to be sure we remove it if it's a third-party icon.

File: src/sass/mixins/_Icon.Mixins.scss [hunk 2]
```diff
@@ -168,6 +169,7 @@
 @mixin ms-Icon--AnalyticsReport { content: $ms-icon-code-AnalyticsReport; }
 @mixin ms-Icon--AnchorLock { content: $ms-icon-code-AnchorLock; }
 @mixin ms-Icon--Annotation { content: $ms-icon-code-Annotation; }
+@mixin ms-Icon--ApacheIvyLogo32 { content: $ms-icon-code-ApacheIvyLogo32; }
```
Comment (mikewheaton) [inline src/sass/mixins/_Icon.Mixins.scss]: Same as the previous comment; this likely needs to be removed.

File: src/sass/variables/_Icon.Variables.scss [hunk 3]
```diff
@@ -62,6 +63,7 @@ $ms-icon-code-AnalyticsQuery: '\F1DF';
 $ms-icon-code-AnalyticsReport: '\F1E1';
 $ms-icon-code-AnchorLock: '\F511';
 $ms-icon-code-Annotation: '\E924';
+$ms-icon-code-ApacheIvyLogo32: '\F5D1';
```
Comment (mikewheaton) [inline src/sass/variables/_Icon.Variables.scss]: Likely needs to be removed.

File: src/sass/variables/_Icon.Variables.scss [hunk 4]
```diff
@@ -1449,6 +1472,7 @@ $ms-icon-map: (
   'AnalyticsReport': $ms-icon-code-AnalyticsReport,
   'AnchorLock': $ms-icon-code-AnchorLock,
   'Annotation': $ms-icon-code-Annotation,
+  'ApacheIvyLogo32': $ms-icon-code-ApacheIvyLogo32,
```
Comment (mikewheaton) [inline src/sass/variables/_Icon.Variables.scss]: Likely needs to be removed.

**Pull request merged**